eng/perf-job.yml

```diff
@@ -0,0 +1,75 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osIdentifier: ''
+  readyToRun: true
+  helixQueues: ''
+  container: ''
+  crossrootfsDir: ''
+  framework: netcoreapp5.0 # Specify the appropriate framework when running release branches (ie netcoreapp3.0 for release/3.0)
+
+### Test job
```

Comment:
This seems like it does not fit as this isn't a test job?

Reply:
Copy paste from the test-job yaml